include/openspace/navigation/orbitalnavigator.h

```diff
@@ -251,7 +253,9 @@ class OrbitalNavigator : public properties::PropertyOwner {
     bool _invertIdleBehaviorInterpolation = false;
 
     IdleBehavior _idleBehavior;
-    float _idleBehaviorTriggerTimer = 0.f;
+    float _idleBehaviorTriggerTimer = 0.0;
```

Comment:
```suggestion
    float _idleBehaviorTriggerTimer = 0.f;
```

`0.f` was correct since it is a `float`